src/main/java/sixgaezzang/sidepeek/users/controller/UserController.java

```diff
@@ -49,9 +51,12 @@ public ResponseEntity<Void> signUp(@RequestBody @Valid SignUpRequest request) {
     }
 
     @GetMapping
+    @Operation(summary = "회원 검색")
+    @ApiResponse(responseCode = "200", description = "회원 검색 성공")
+    @Parameter(name = "keyword", description = "검색어", example = "sixgaezzang6")
     public ResponseEntity<UserSearchResponse> searchByNickname(
         @RequestParam(required = false)
-        @Size(max = 20, message = "최대 20자의 키워드로 검색할 수 있습니다.")
+        @Size(max = MAX_NICKNAME_LENGTH, message = "최대 20자의 키워드로 검색할 수 있습니다.")
```

Comment:
```suggestion
        @Size(max = MAX_NICKNAME_LENGTH, message = "최대 " + MAX_NICKNAME_LENGTH + "자의 키워드로 검색할 수 있습니다.")
```
위 SkillController 처럼 UserController도 이와 같이 통일시켜주는 건 어떻게 생각하시나요..?

Reply:
어머 매의 눈!! 감사합니다! 🥰